sdk/servicebus/service-bus/package.json

```diff
@@ -52,7 +52,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --timeout 120000 --full-trace dist-esm/test/*.spec.js",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --timeout 120000 --exit --full-trace dist-esm/test/*.spec.js",
```

Comment:
https://boneskull.com/mocha-v4-nears-release/#mochawontforceexit

Reply:
Mocha is not exiting even though all the tests complete, this is making the pipeline take more than 3hrs to finish the integration-test:node step, `--exit` flag helps close the mocha process once the tests are completed.

Reply:
Logged an issue so that we don't miss - https://github.com/Azure/azure-sdk-for-js/issues/7720